systemvm/patches/debian/config/opt/cloud/bin/cs/CsFile.py

```diff
@@ -35,10 +35,9 @@ def load(self):
                 self.new_config.append(line)
         except IOError:
             logging.debug("File %s does not exist" % self.filename)
-            return
         else:
             logging.debug("Reading file %s" % self.filename)
-            self.config = copy.deepcopy(self.new_config)
+            self.config = list(self.new_config)
```

Comment:
copy.deepcopy may be a necessary evil, if the list new_config contains objects. I wrote an example to test this: https://gist.github.com/bhaisaab/fe85fb02e1a7fda2fd2d